packages/trader/src/Modules/MT5/Containers/mt5-password-manager-modal.jsx

```diff
@@ -33,7 +33,7 @@ class MT5PasswordManagerModal extends React.Component {
         },
     };
 
-    shouldComponentUpdate(next_props, next_state) {
+    componentDidUpdate(next_props, next_state) {
```

Comment:
It was a part of an incomplete refactor. `shouldComponentUpdate` should not be present in this file, since it is already a `connect`ed component and mobx handles propchange detection for us. So, I noticed there is state mutation after component update, hence I changed it to `componentDidUpdate` but I forgot to remove the boolean return part. thanks for noticing.